figures/serializers.py

```diff
@@ -740,12 +741,19 @@ class Meta:
         model = get_user_model()
         editable = False
         fields = (
-            'id', 'username', 'name', 'email', 'country', 'is_active', 'course_activity_date',
-            'year_of_birth', 'level_of_education', 'gender', 'date_joined', 'last_login',
-            'bio', 'courses', 'language_proficiencies', 'profile_image',
+            'id', 'username', 'name', 'email', 'country', 'is_active',
+            'course_activity_date', 'year_of_birth', 'level_of_education',
+            'gender', 'date_joined', 'last_login', 'bio', 'courses',
+            'language_proficiencies','profile_image', 'registration_fields',
             )
         read_only_fields = fields
 
+    def get_registration_fields(self, user):
```

Comment:
Are we getting all registration field (optional + required) or just the required registration fields?

Reply:
Only `required` currently, should we add `optional` as well?

Reply:
No. We only need required registration fields.